Modelica/Electrical/Analog/Interfaces.mo

```diff
@@ -544,9 +490,6 @@ The device temperature <strong>internalHeatPort.T</strong> can be used to descri
     extends OnePort;
     parameter SI.Current offset=0 "Current offset";
     parameter SI.Time startTime=0 "Time offset";
-    replaceable Modelica.Blocks.Interfaces.SignalSource signalSource(
-        final offset = offset, final startTime=startTime) annotation (Placement(
-          transformation(extent={{70,69},{91,89}})));
```

Comment:
That change seems really good, but unrelated to the PR.
How did the signalSource appear?

Reply:
Re-introuced by c4d6d1bbc65306e69062a02bf4d485e1018e6a5e